sdk/core/azure-core-tracing-opencensus/src/test/java/com/azure/core/tracing/opencensus/OpenCensusTracerTest.java

```diff
@@ -35,13 +79,119 @@ public void startSpanNullPointerException() {
     }
 
     @Test
-    public void startSpanReturnsNewSpanTest() {
+    public void startSpanParentContextFlowTest() {
+        // Arrange
+        final SpanId parentSpanId = parentSpan.getContext().getSpanId();
+
+        // Act
+        final Context updatedContext = openCensusTracer.start(METHOD_NAME, tracingContext);
+
+        // Assert
+        assertSpanWithExplicitParent(updatedContext, parentSpanId);
+        final RecordEventsSpanImpl recordEventsSpan =
+            (RecordEventsSpanImpl) updatedContext.getData(PARENT_SPAN_KEY).get();
+        Assert.assertNull(recordEventsSpan.getKind());
```

Comment:
just curious, why check for null kind?

Reply:
Just to validate it isn't getting set anywhere in the workflow.